buildkite/culprit_finder.py

```diff
@@ -57,8 +57,9 @@ def get_bazel_commits_between(first_commit, second_commit):
 
 
 def get_configs(project_name):
-    http_config = bazelci.DOWNSTREAM_PROJECTS[project_name]["http_config"]
-    configs = bazelci.fetch_configs(http_config, None)
+    http_config = bazelci.DOWNSTREAM_PROJECTS[project_name].get("http_config", None)
```

Comment:
Here and below:
Nit: you can drop the "None" since that's already the default value.

Reply:
Thanks!